package.json

```diff
@@ -71,5 +72,9 @@
     "./**/*.{js,jsx,ts,tsx,scss}": [
       "npm run format"
     ]
+  },
+  "volta": {
```

Comment:
things look good to me just one addition please mention the installation steps in the Readme file using volta.

Reply:
Sure will do that.